VAMobile/src/screens/HealthScreen/SecureMessaging/EditDraft/EditDraft.test.tsx

```diff
@@ -266,75 +234,52 @@ context('EditDraft', () => {
     })
 
     it('should display an AlertBox', async () => {
-      await waitFor(() => {
-        expect(testInstance.findAllByType(AlertBox).length).toEqual(1)
-      })
+      expect(screen.getByText("We can't match you with a provider")).toBeTruthy()
     })
 
     describe('on click of the go to inbox button', () => {
       it('should call useRouteNavigation and updateSecureMessagingTab', async () => {
-        await waitFor(() => {
-          testInstance.findByProps({ label: 'Go to inbox' }).props.onPress()
-          expect(navigateSpy).toHaveBeenCalled()
-          expect(updateSecureMessagingTab).toHaveBeenCalled()
-        })
+        fireEvent.press(screen.getByText('Go to inbox'))
+        expect(navigateSpy).toHaveBeenCalled()
+        expect(updateSecureMessagingTab).toHaveBeenCalled()
       })
     })
   })
 
   describe('when hasLoadedRecipients is false', () => {
     it('should display the LoadingComponent', async () => {
-      await waitFor(() => {
-        initializeTestInstance({ loading: true })
-        expect(testInstance.findAllByType(LoadingComponent).length).toEqual(1)
-      })
+      initializeTestInstance({ loading: true })
+      expect(screen.getByText("Loading your draft...")).toBeTruthy()
     })
   })
 
   describe('when there is an error', () => {
     it('should display the ErrorComponent', async () => {
-      await waitFor(() => {
-        initializeTestInstance({ screenID: ScreenIDTypesConstants.SECURE_MESSAGING_COMPOSE_MESSAGE_SCREEN_ID })
-        expect(testInstance.findAllByType(ErrorComponent).length).toEqual(1)
-      })
-    })
-  })
-
-  describe('on click of the crisis line banner', () => {
-    it('should call useRouteNavigation', async () => {
-      await waitFor(() => {
-        testInstance.findAllByType(TouchableWithoutFeedback)[1].props.onPress()
-        expect(navigateToVeteransCrisisLineSpy).toHaveBeenCalled()
-      })
+      initializeTestInstance({ screenID: ScreenIDTypesConstants.SECURE_MESSAGING_COMPOSE_MESSAGE_SCREEN_ID })
+      expect(screen.getByText("The app can't be loaded.")).toBeTruthy()
     })
   })
 
   describe('on click of the collapsible view', () => {
     it('should show the Reply Help panel', async () => {
-      await waitFor(() => {
-        testInstance.findByProps({ accessibilityLabel: 'Only use messages for non-urgent needs' }).props.onPress()
-      })
-      expect(navigateToReplyHelpSpy).toHaveBeenCalled()
+      fireEvent.press(screen.getByLabelText('Only use messages for non-urgent needs'))
```

Comment:
Thanks for improving a bunch of unit tests!

Reply:
no problem lol